lib/modulesync.rb

```diff
@@ -173,8 +175,14 @@ def self.update(cli_options)
     managed_modules.each do |puppet_module|
       begin
         manage_module(puppet_module, module_files, defaults)
-      rescue # rubocop:disable Lint/RescueWithoutErrorClass
-        warn "Error while updating '#{puppet_module.given_name}'"
+      rescue ModuleSync::Error, Git::GitExecuteError => e
+        message = e.message || "Error during '#{options[:command]}'"
+        message = "#{puppet_module.given_name}: #{message}"
+        $stderr.puts message
```

Comment:
This may be my taste, but I dislike assigning variables again. Perhaps this is just as good?
```suggestion
        $stderr.puts "#{puppet_module.given_name}: #{message}"
```

Reply:
LGTM!

Reply:
Done.